### ISSUE TITLE
company slugs

### ISSUE THREAD

**Issue by gemantzu** [beam-community/elixir-companies#588]
It seems that some people are messing with the urls, trying to find their company. I see some errors in the form of `/en/companies/CompanyName`. Should we add a slug field for the company name or should we just try to find the company based on the url followed?

**Comment by doomspork** [comment]
Yeah. We need to support slugs. We get hundreds of hits of people trying to use them.
